### DISCUSSION TITLE
Bluetooth audio

### DISCUSSION THREAD

**Discussion by Xk-w** [schreibfaul1/ESP32-audioI2S#576] [General]
Does the ESP32-audioi2s library support the mode of receiving audio as a Bluetooth device? Similar to a Bluetooth audio decoding chip!

**Comment by schreibfaul1** [comment]
This library does not have Bluetooth functionality. BT sink/source can be added, however, as was done in the [ESPuino](https://github.com/biologist79/ESPuino) project, for example.
